server/src-lib/Hasura/RQL/DML/RemoteJoin.hs

```diff
@@ -420,7 +420,12 @@ fetchRemoteJoinFields env manager reqHdrs userInfo remoteJoins = do
                                  (map _rjfField batchList)
         gqlReqUnparsed = (GQLQueryText . G.renderExecutableDoc . G.ExecutableDocument . unGQLExecDoc) <$> gqlReq
     -- NOTE: discard remote headers (for now):
-    (_, _, respBody) <- execRemoteGQ' env manager userInfo reqHdrs gqlReqUnparsed rsi G.OperationTypeQuery
+    (_, _, respBody) <- do
+      -- TODO(evertedsphere): move this into metadata, so that a trace further
```

Comment:
I'm not following this note. We can't put this in metadata here, or it'll appear at the wrong place in the trace.

Reply:
right, yeah, i put this note in a while ago so i can't say what i was thinking then...